__tests__/app.test.js

```diff
@@ -157,4 +157,53 @@ describe("GET", () => {
 
   })
 
+  //task 6
+  describe("api/articles/:article_id/comments", () => {
+
+    
+    //error handling
+
+    //400:
+
+    it("200: should respond with array of comments for given article_id, with each article comment to have 6 matching properties", () => {
+      return request(app).get("/api/articles/3/comments").expect(200).then((result) => {
+        console.log(result.body.comment, "in test")
```

Comment:
Make sure to remove your console.logs, and any time you loop over an array in a test, ensure that the array is not empty beforehand! This is because the expects will only run if the forEach runs - and if the array is empty, it won't run :)